sourcegraph/sourcegraph
# Improved CTAs

This closes #43752 with some improved CTAs.

## Changelog
- added a `Try Sourcegraph Cloud` CTA button for logged in dotCom and self-hosted users
- moved the Feedback button from the top nav to the user menu as a modal instead of a popup for auth'd users. Unauth'd users will still see the Feedback button as a popup option.
- no change to pro tip since it's already shown to all users
- instead of adding a CTA to our settings page (without being spammy since we already have a button in the top nav), I added it to the subscriptions page since we mention self-hosted pricing there

There is no way to know a self-hosted user vs a dot com user, so consolidating the wording for one option simplifies the solution.

## Test plan
- check that the feedback prompt in the modal works from the user menu
- check that the global nav cta button works and links to signup
- check that the signup cta on the pricing/subs page links to signup

## App preview:

- [Web](https://sg-web-brett-ctas.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
